lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +74,47 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        file = json.load(open(self._path_to_config))
+
+        config = ConfigDTO(
+            seed_urls=file['seed_urls'],
+            headers=file['headers'],
+            total_articles_to_find_and_parse=file['total_articles_to_find_and_parse'],
+            encoding =file['encoding'],
+            timeout=file['timeout'],
+            should_verify_certificate=file['should_verify_certificate'],
+            headless_mode=file['headless_mode']
+        )
+        return config
+
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        config = json.load(open(self._path_to_config))
+        if not isinstance(config['seed_urls'], list) or \
```

Comment:
we do not work with any website, right? we need seed urls to be exactly of the website we target

Reply:
Should I add more conditions for this? There were no such info in the documentation

Reply:
it is logic. you can't work with any site. you specialize on it, therefore, all other websites should be rejected - a.k.a raised Exception

Reply:
done